ecs-cli/modules/clients/aws/ecs/client_test.go

```diff
@@ -607,6 +607,54 @@ func TestRunTask_WithTaskNetworking(t *testing.T) {
 	assert.NoError(t, err, "Unexpected error when calling RunTask")
 }
 
+func TestRunTask_WithTaskPlacement(t *testing.T) {
```

Comment:
Just something to think about (not a blocker for merging): Now that you've refactored the ECS Client's RunTask to just take the RunTaskInput and then call the SDK's RunTask, do we really need tests like this? This test just passes in the 2 new things through the RunTaskInput object and then verifies that the client was called with the same inputs. One might say that it never hurts to have more tests... but I disagree because the more tests you have, the more code you have to change during future refactorings/code changes.

Reply:
I definitely agree that tests should be treated as first-class citizens for purposes of code maintenance; in this case however, given our mocking strategy I think this is still a useful test to ensure that we are passing the right inputs through to our ECS Client (and that certain fields are *not* passed through in certain scenarios, to ensure we get the expected defaults that ECS provides.